### PR TITLE
Update FAQ: Add infobox for reduced functionality

### PR DESCRIPTION
**Pull request by larswmh**
This PR adds the blue info box to 213 FAQ entries, as the Corona-Warn-App functionality is being heavily reduced. Thus, every FAQ entry that covers a soon to be disabled functionality, or is related to it in any way, received the blue box. 

While reviewing, you might come across some changes that are not really related to the opening comment, but were practical to implement with this massive batch of changes all together. This includes:
- Moving `#multiple_exposure_checks` and `#ENError5` to match their placement across both languages
- Some "UPDATED" / "Updated on xx.xx.xxxx" texts were removed and multiple boxes in a single entry have been merged into a single box, seperating information with a `<hr />` to avoid confusion.
- The info box about G-statuses on `#admission_policy` was also added to `#admission_policy_2gplus`, `#admission_policy_2gplus_booster`, `#admission_policy_2gplus_booster_howto` and `#admission_policy_2gplus_not_shown`
-  The test infrastructure shutdown info box was added to `#dashboard_pcr_tests` 
- `#update_faq` was modified to represent the date of when this change will take effect

---
Internal Tracking ID: [EXPOSUREAPP-13612](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13612)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -341,7 +347,7 @@
                                 "anchor": "risk_update_24",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated March 30, 2022</b>",
+                                    "<div class='update'>The information in this FAQ entry is related to features or describes features of the app that are no longer available as of May 1, 2023, and is therefore no longer up-to-date. For more information, click here: <a href='#ramp_down' target='_blank'>What is the background for the end of the development of the CWA?</a><hr />The information in this FAQ entry is related to features or describes features of the app that are no longer available as of May 1, 2023, and is therefore no longer up-to-date. For more information, click here: <a href='#ramp_down' target='_blank'>What is the background for the end of the development of the CWA?</a></div>",
```
Comment (MikeMcC399) [inline src/data/faq.json:350]: This has been added twice.

File: src/data/faq.json [hunk 2]
```diff
@@ -1501,8 +1526,7 @@
...
+                                    "<div class='update'>The information in this FAQ entry is related to features or describes features of the app that are no longer available as of May 1, 2023, and is therefore no longer up-to-date. For more information, click here: <a href='#ramp_down' target='_blank'>What is the background for the end of the development of the CWA?</a><hr />As of version 2.27, the Corona-Warn-App (CWA) displays the current mask status as a dedicated symbol in the \"Certificates\" area if any federal states define their own more extensive mask obligations.<br>According to the fall/winter plan of the federal government this is divided into the \"federal legislations\" and \"supplemental measures of the federal states\" (cf. sources below).<br><b>Currently, there are no additional measures of the states.</b><hr/>",
```
Comment (MikeMcC399) [inline src/data/faq.json:1529]: "fall/winter plan" this is dated and it is unclear what year this belongs to

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I skimmed only through the English version and made a couple of individual comments.

The changes don't fit [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) (Don't Repeat Yourself) principles and are almost impossible to review carefully and to maintain. It would be better to look for ways to define boilerplate blocks of texts which are maintained in one place only and inserted by reference where needed.

File: src/data/faq.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq.json]: Added the year in https://github.com/corona-warn-app/cwa-website/pull/3474/commits/3420f6ef18b4aab32751f66db3cda39b99703297, thanks!

File: src/data/faq.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq.json]: Removed the duplicate in https://github.com/corona-warn-app/cwa-website/pull/3474/commits/3420f6ef18b4aab32751f66db3cda39b99703297, thanks!

**Comment by MikeMcC399** [comment]
- Is there a benefit from keeping the out-of-date texts visible? Have you considered deleting FAQ articles which are no longer applicable?

- Also, as seen by issue https://github.com/corona-warn-app/cwa-website/issues/3475 external links will deteriorate over time. It seems that other "webmasters" feel no responsibility for maintaining links to obsolete content and simply delete pages, leaving an HTTP `404 not found` error behind. The more content (with links) you leave online, the more maintenance you will need to do to keep the weekly link tests from not failing.

**Comment by larswmh** [comment]
@MikeMcC399 

> * Is there a benefit from keeping the out-of-date texts visible? Have you considered deleting FAQ articles which are no longer applicable?

Deleting FAQ entries was taken into consideration, but keeping them all was preferred to keep the site as a sort of archive now that the app is slowly shutting down. If anyone is interested, they will be able to look through everything that has been possible with the app in the future.

> * Also, as seen by issue [Broken link zusammengegencorona wann-muss-ich-in-quarantaene-und-wann-nicht #3475](https://github.com/corona-warn-app/cwa-website/issues/3475) external links will deteriorate over time. It seems that other "webmasters" feel no responsibility for maintaining links to obsolete content and simply delete pages, leaving an HTTP `404 not found` error behind. The more content (with links) you leave online, the more maintenance you will need to do to keep the weekly link tests from not failing.

That is correct, but I don't think it's that much of an issue, especially if there's a box above the FAQ stating that the content is outdated. For the time being, we will still actively monitor this repository, and the automated weekly check for broken links really helps us with that.

**Comment by MikeMcC399** [comment]
@larswmh 

Thank you for explaining why you want to keep the contents. Of course it is your decision about this. I would have preferred the non-active contents to be removed instead of hoarding it. (There is a web wayback machine which serves the interests of those looking for historical contents e.g. https://web.archive.org/web/20230327123134/https://www.coronawarn.app/en/.)

I noticed that the test `faq_link_attr` failed a couple of times. I could not find the reason for this and it is now working again. It was only failing on GitHub, not locally. You may need to keep an eye on this. Due to this PR it is now checking around 700 links in the each language main FAQ section. This is ~40% more than before.

**Comment by MikeMcC399** [comment]
@larswmh 

There is no longer a "Settings" option in the CWA menu, so any FAQ article, such as https://www.coronawarn.app/en/faq/results/#notification_settings which includes instructions to tap on "Settings" is now obsolete.

**Comment by larswmh** [comment]
@MikeMcC399 

> There is no longer a "Settings" option in the CWA menu, so any FAQ article, such as https://www.coronawarn.app/en/faq/results/#notification_settings which includes instructions to tap on "Settings" is now obsolete.

Thanks for the information, I've added the box to #notification_settings in https://github.com/corona-warn-app/cwa-website/pull/3474/commits/56670c124eea317928e07f12f6fcbc1cf2f038b9

**Pull request merged**